code/modules/admin/ticket.dm

```diff
@@ -8,14 +8,21 @@ var/list/ticket_panels = list()
 	var/list/msgs = list()
 	var/datum/client_lite/closed_by
 	var/id
+	var/sql_id
 	var/opened_time
 	var/timeout = FALSE
+	var/to_check
+	var/to_close
 
 /datum/ticket/New(var/datum/client_lite/owner)
 	src.owner = owner
 	tickets |= src
 	id = tickets.len
 	opened_time = world.time
+	if(establish_db_connection())
+		var/sql_ckey = sanitizeSQL(owner.ckey)
+		var/DBQuery/ticket_query = dbcon.NewQuery("INSERT INTO erro_admin_tickets(ckey, round, inround_id, status, open_date) VALUES ('[sql_ckey]', '[game_id]', [src.id], 'OPEN', NOW());")
```

Comment:
Need backticks around tables/columns.
Also remove the erro, that's a old person's tag when they created databases.
INSERT INTO \`admin_tickets\` (\`ckey\`, \`round\`, \`inround_id\`, \`status\`, \`open_date\`)

Reply:
We discussed removing `erro` on Discord, it may make the table look weird and out place since literally every other table uses it at this point.

Reply:
kk

Reply:
@Andrew-Fall check pls